bin/stacks/routing-api-stack.ts

```diff
@@ -327,15 +327,12 @@ export class RoutingAPIStack extends cdk.Stack {
       evaluationPeriods: 3,
     })
 
-    // Simulations can fail for valid reasons. For example, if the simulation reverts due
-    // to slippage checks (can happen with FOT tokens sometimes since our quoter does not
-    // account for the fees taken during transfer when we show the user the quote).
-    //
-    // For this reason we only alert on SEV3 to avoid unnecessary pages.
-    const simulationAlarmSev3 = new aws_cloudwatch.Alarm(this, 'RoutingAPI-SEV3-Simulation', {
-      alarmName: 'RoutingAPI-SEV3-Simulation',
+    // Tenderly sim system downtime is sev2, because it's swap blocking from trading-api.
+    // We have confidence that tenderly is down
+    const simulationAlarmSev2 = new aws_cloudwatch.Alarm(this, 'RoutingAPI-SEV2-Simulation', {
+      alarmName: 'RoutingAPI-SEV2-Simulation',
       metric: new MathExpression({
-        expression: '100*(simulationFailed/simulationRequested)',
```

Comment:
we don't want to trigger on `simulationFailed` anymore?

Reply:
good question, we no longer should alert off `simulationFailed`, because originally it's the mix of both tenderly system downtime, and on-chain simulated transaction failure. Now we just want to be alerted on the tenderly system down, which is action-able. on-chain simulated transaction failures, otoh, is not action-able to us.